online-shop/src/components/listProducts/listProducts.component.tsx

```diff
@@ -0,0 +1,29 @@
+import { Grid } from "@material-ui/core";
+import React from "react";
+import Product, {Produs} from "../product/product.component"
+import useListProductStyles from "./listProducts.styles";
+
+
+const productsList: Array<Produs>= 
```

Comment:
I'd rather you kept your mock data within the service such that you can maintain the intended flow of data.